[2024-05-18T23:52:37.943+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-18T23:52:40.175+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Real_Estate_Webscraping_ETL.Process_and_Send_Data_to_DB scheduled__2024-05-18T12:06:00+00:00 [queued]>
[2024-05-18T23:52:41.004+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Real_Estate_Webscraping_ETL.Process_and_Send_Data_to_DB scheduled__2024-05-18T12:06:00+00:00 [queued]>
[2024-05-18T23:52:41.160+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 6
[2024-05-18T23:52:42.806+0000] {taskinstance.py:2330} INFO - Executing <Task(SparkSubmitOperator): Process_and_Send_Data_to_DB> on 2024-05-18 12:06:00+00:00
[2024-05-18T23:52:43.509+0000] {standard_task_runner.py:63} INFO - Started process 272 to run task
[2024-05-18T23:52:43.722+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Real_Estate_Webscraping_ETL', 'Process_and_Send_Data_to_DB', 'scheduled__2024-05-18T12:06:00+00:00', '--job-id', '384', '--raw', '--subdir', 'DAGS_FOLDER/webscrapping.py', '--cfg-path', '/tmp/tmpv8w3h0fi']
[2024-05-18T23:52:44.202+0000] {standard_task_runner.py:91} INFO - Job 384: Subtask Process_and_Send_Data_to_DB
[2024-05-18T23:52:55.100+0000] {task_command.py:426} INFO - Running <TaskInstance: Real_Estate_Webscraping_ETL.Process_and_Send_Data_to_DB scheduled__2024-05-18T12:06:00+00:00 [running]> on host 4801ac514ca9
[2024-05-18T23:53:10.835+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='svc' AIRFLOW_CTX_DAG_ID='Real_Estate_Webscraping_ETL' AIRFLOW_CTX_TASK_ID='Process_and_Send_Data_to_DB' AIRFLOW_CTX_EXECUTION_DATE='2024-05-18T12:06:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-18T12:06:00+00:00'
[2024-05-18T23:53:11.816+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-18T23:53:16.284+0000] {base.py:84} INFO - Using connection ID 'spark_default' for task execution.
[2024-05-18T23:53:16.451+0000] {spark_submit.py:401} INFO - Spark-Submit cmd: spark-submit --master spark://spark-master:7077 --packages org.apache.spark:spark-sql-kafka-0-10_2.12:3.5.1,com.datastax.spark:spark-cassandra-connector_2.12:3.5.0,org.apache.kafka:kafka-clients:3.5.1,commons-logging:commons-logging:1.2 --name arrow-spark --deploy-mode client dags/includes/jobs/spark_consumer.py
[2024-05-18T23:53:20.200+0000] {spark_submit.py:571} INFO - /home/***/.local/lib/python3.11/site-packages/pyspark/bin/load-spark-env.sh: line 68: ps: command not found
[2024-05-18T23:55:56.165+0000] {local_task_job_runner.py:124} ERROR - Received SIGTERM. Terminating subprocesses
[2024-05-18T23:55:57.955+0000] {process_utils.py:132} INFO - Sending 15 to group 272. PIDs of all processes in the group: [524, 272]
[2024-05-18T23:55:57.977+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 272
[2024-05-18T23:55:58.356+0000] {taskinstance.py:2611} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-05-18T23:55:58.980+0000] {spark_submit.py:697} INFO - Sending kill signal to spark-submit
[2024-05-18T23:55:59.052+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-18T23:56:00.350+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self.application)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 490, in submit
    self._process_spark_submit_log(iter(self._submit_sp.stdout))  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 539, in _process_spark_submit_log
    for line in itr:
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 2613, in signal_handler
    raise AirflowTaskTerminated("Task received SIGTERM signal")
airflow.exceptions.AirflowTaskTerminated: Task received SIGTERM signal
[2024-05-18T23:56:00.621+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=Real_Estate_Webscraping_ETL, task_id=Process_and_Send_Data_to_DB, run_id=scheduled__2024-05-18T12:06:00+00:00, execution_date=20240518T120600, start_date=20240518T235240, end_date=20240518T235600
[2024-05-18T23:56:02.520+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=524, status='terminated', started='23:53:16') (524) terminated with exit code None
[2024-05-18T23:56:02.560+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=272, status='terminated', exitcode=2, started='23:52:42') (272) terminated with exit code 2
[2024-05-18T23:56:02.704+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 143
[2024-05-18T23:56:03.555+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-18T23:56:04.490+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
